core/src/main/java/tech/pegasys/eth2signer/core/Runner.java

```diff
@@ -278,6 +279,9 @@ private HttpServer createServerAndWait(
         new HttpServerOptions()
             .setPort(config.getHttpListenPort())
             .setHost(config.getHttpListenHost())
+            .setTcpKeepAlive(false)
```

Comment:
should this be configurable or these defaults are reasonable?

Reply:
oops - configurable.